packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -158,10 +164,69 @@ describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
         await deployContract(mediumSizeContract, accounts[0].wallet);
 
         const remainingHbarsAfter = Number(await metrics.get(testConstants.METRICS.REMAINING_HBAR_LIMIT));
-        const expectedCost = 354819247;
+        const expectedCost = 331926551;
         verifyRemainingLimit(expectedCost, remainingHbarsBefore, remainingHbarsAfter);
       });
 
+      it('HBAR limiter is updated within acceptable tolerance range in relation to actual spent amount by the relay operator', async function () {
+        const TOLERANCE = 0.01;
+
+        await new Promise((r) => setTimeout(r, 3000));
+        const operatorAccount = process.env.OPERATOR_ID_MAIN;
+        const remainingHbarsBefore = Number(await metrics.get(testConstants.METRICS.REMAINING_HBAR_LIMIT));
+        expect(remainingHbarsBefore).to.be.gt(0);
+        const operatorBalanceBefore = (await mirrorNode.get(`/accounts/${operatorAccount}`, requestId)).balance.balance;
+
+        const largeContract = await Utils.deployContract(
+          largeContractJson.abi,
+          largeContractJson.bytecode,
+          accounts[0].wallet,
+        );
+        await largeContract.waitForDeployment();
+        expect(largeContract.target).to.not.be.null;
```

Comment:
We can reuse the method `deployContract` inside `hbarLimiter.spec.ts` which already does the same checks:
```suggestion
        const largeContract = await deployContract(largeContractJson, accounts[0].wallet);
```